jrebel/pom.xml

```diff
@@ -35,19 +35,4 @@
     </dependency>
   </dependencies>
 
-  <!--
```

Comment:
This one isn't being used (commented out). The URL doesn't exist. And is http. Seems best to just remove this section.